### PR TITLE
Shadergraph/documentation/sg-doc-2592

### PR DESCRIPTION
**Pull request by ValGrimm-U3D**
# **Please read the [Contributing guide](CONTRIBUTING.md) before making a PR.**

* Read the [Graphics repository & Yamato FAQ](http://go/graphics-yamato-faq).

    [Y ] Have you added a backport label (if needed)? For example, the need-backport-* label. After you backport the PR, the label changes to backported-*.
    [Y ] Have you updated the changelog? Each package has a CHANGELOG.md file.
    [ Y] Have you updated or added the documentation for your PR? When you add a new feature, change a property name, or change the behavior of a feature, it's best practice to include related documentation changes in the same PR. If you do add documentation, make sure to add the relevant Graphics Docs team member as a reviewer of the PR. If you are not sure which person to add, see the Docs team contacts sheet.
    [N/A ] Have you added a graphic test for your PR (if needed)? When you add a new feature, or discover a bug that tests don't cover, please add a graphic test.

Purpose of this PR

Why is this PR needed, what hard problem is it solving/fixing?
Correcting Blackboard docs

---
### Testing status
Describe what manual/automated tests were performed for this PR
N/A

---
### Comments to reviewers
Notes for the reviewers you have assigned.

### CODE REVIEW

File: com.unity.shadergraph/CHANGELOG.md [hunk 1]
```diff
@@ -8,11 +8,13 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Added
   - Stereo Eye Index, Instance ID, and Vertex ID nodes added to the shadergraph library.
+  - Added information about selecting and unselecting items to the Blackboard article.
 
 ### Changed
 - Change Asset/Create/Shader/Blank Shader Graph to Asset/Create/Shader Graph/Blank Shader Graph
 - Change Asset/Create/Shader/Sub Graph to Asset/Create/Shader Graph/Sub Graph
 - Change Asset/Create/Shader/VFX Shader Graph to Asset/Create/Shader Graph/VFX Shader Graph
+- Adjusted Blackboard article to solve DOC-2592
```
Comment (marctem) [inline com.unity.shadergraph/CHANGELOG.md:17]: The Jira ticket number is not useful in the changelog, please describe the actual issue you're solving, and optionally add a link to a public IssueTracker card (if there is one).

**Review by marctem — CHANGES_REQUESTED** [review]

File: com.unity.shadergraph/CHANGELOG.md (hunk 1 above)
Comment (ValGrimm-U3D) [reply com.unity.shadergraph/CHANGELOG.md:17]: oops, had fixed that on the last PR of this branch and then somehow it got unfixed, sorry.

**Review by Nightmask3 — APPROVED** [review]
Looks good!

**Comment by ValGrimm-U3D** [comment]
Hi @marctem , I re-did the changelog fix, can you approve?

**Review by marctem — APPROVED** [review]

**Comment by ValGrimm-U3D** [comment]
@Nightmask3 can you help me clear the failed checks on this one?

**Comment by Nightmask3** [comment]
@ValGrimm-U3D looks like we're good to go! Hit the big green button whenever you're ready 😄 
Good stuff!

**Pull request merged**